### PR TITLE
[Bridge] [Twig] replaces tabs by spaces in Twig form themes templates.

### PR DESCRIPTION
**Pull request by hhamon**
| Q | A |
| --- | --- |
| Branch? | master |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | ~ |
| License | MIT |
| Doc PR | ~ |

### CODE REVIEW

**Comment by lyrixx** [comment]
:+1:

File: src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig [hunk 1]
```diff
@@ -89,20 +89,20 @@
 
 {% block dateinterval_widget %}
```
Comment (stof) [inline src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig]: we also have some missing whitespace control between the opening of the block and the if tag. Can you add it, like in time_widget ?

File: src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig [hunk 2]
```diff
@@ -89,20 +89,20 @@
...
     {% if widget == 'single_text' %}
-	{{- block('form_widget_simple') -}}
+        {{- block('form_widget_simple') -}}
     {% else %}
```
Comment (stof) [inline src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig]: we need the access control here too

File: src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig [hunk 3]
```diff
@@ -89,20 +89,20 @@
...
+        <div {{ block('widget_container_attributes') }}>
+            {{ form_errors(form) }}
+            {% if with_years %}{{ form_widget(form.years) }}{% endif %}
+            {% if with_months %}{{ form_widget(form.months) }}{% endif %}
+            {% if with_weeks %}{{ form_widget(form.weeks) }}{% endif %}
+            {% if with_days %}{{ form_widget(form.days) }}{% endif %}
+            {% if with_hours %}{{ form_widget(form.hours) }}{% endif %}
+            {% if with_minutes %}{{ form_widget(form.minutes) }}{% endif %}
+            {% if with_seconds %}{{ form_widget(form.seconds) }}{% endif %}
+            {% if with_invert %}{{ form_widget(form.invert) }}{% endif %}
```
Comment (stof) [inline src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig]: all these should use whitespace control as well

**Comment by hhamon** [comment]
@stof @lyrixx I've made the requested changes ;)

**Comment by dunglas** [comment]
👍

**Comment by fabpot** [comment]
Thank you @hhamon.

**Pull request merged**